ext/standard/tests/serialize/bug49649.phpt

```diff
@@ -33,7 +33,12 @@ class Foo
 $class = unserialize(base64_decode($serialized));
 var_dump($class);
 ?>
---EXPECT--
+--EXPECTF--
+Deprecated: Creation of dynamic property Foo::$ is deprecated in %s on line %d
```

Comment:
https://github.com/igbinary/igbinary/pull/362

https://github.com/igbinary/igbinary/pull/362/files#diff-57bd563fa14d413e9e63720be6c9bc10ee7a72fe87b2936938f6d8be467f2c0e zend_get_unmangled_property_name seems useful for `unserialize()` in avoiding the truncation on null bytes for `Foo::$ is`

(note: it emits notices for invalid user-provided property name data, but we're already emitting notices, so that seems fine to me)